bsp/nrf5x/nrf52840/board/board.h

```diff
@@ -5,6 +5,16 @@
 
 #include "nrf.h"
 
+#define MCU_FLASH_PAGE_SIZE 0x1000
+
+#define MCU_FLASH_START_ADRESS       ((uint32_t)0x00000000)
```

Comment:
Or is it better to define the config in Kconfig?

Reply:
I think this is not variable for config. This is const. Do you think const feature in Kconfig is suitable？